components/src/core/EmptyState/EmptyState.tsx

```diff
@@ -1,19 +1,26 @@
-import React, { HTMLAttributes, ReactNode } from 'react';
-import makeStyles from '@mui/styles/makeStyles';
-import { useTheme, Theme } from '@mui/material/styles';
+import React, { ReactNode } from 'react';
+import { styled } from '@mui/material/styles';
+import { cx } from '@emotion/css';
```

Comment:
just a learning moment here but should this import be both cx and css?
import { cx, css } from '@emotion/css'